### DISCUSSION TITLE
Packages: npm view does not show all fields from package.json

### DISCUSSION THREAD

**Discussion by tremppu** [community/community#27463] [Actions]
Hi! I have a private package in GitHub Packages registry and it has few custom fields in package.json but [`npm view`](https://docs.npmjs.com/cli/v7/commands/npm-view) won't return any of them. Example of a custom field like this would be `ng-update` in [@nrwl/next's package.json](https://github.com/nrwl/nx/blob/master/packages/next/package.json#L28) which you can get with `npm view @nrwl/next ng-update`. 

Any ideas why this is not working?

**Comment by lucasvieirasilva** [comment]
Same issue here
